cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/plugin/MacroParser.java

```diff
@@ -146,17 +163,21 @@ private int getEndIndex(String str, int startIndex) {
   private int getArgsStartIndex(String str, String macroStr) {
     int argsStartIndex = macroStr.indexOf('(');
     while (escaped(argsStartIndex, str)) {
```

Comment:
not from this PR, but lets rename to 'isEscaped'. Makes it a little easier to read.